### PR TITLE
Rename da fns

### PR DESCRIPTION
**Pull request by kpp**
# Description

Refactor to reuse common code and reduce boilerplate.

### CODE REVIEW

File: crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs [hunk 1]
```diff
@@ -64,6 +69,14 @@ pub enum DaDataBatchProof {
     // ForcedTransaction(ForcedTransaction),
 }
 
+/// Which type of tx we operate on in DaVerifier
+pub enum DaNamespace {
+    /// BatchProof
+    BatchProof,
+    /// LightClient
+    LightClient,
```
Comment (eyusufatik) [inline crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs]: what about ToBatchProof and ToLightClientProof ??

**Comment by eyusufatik** [comment]
Everything looks good, just need to change the namespace names imo

File: crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs (hunk 1 above)
Comment (kpp) [reply crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs]: Ok, I will

File: crates/primitives/src/constants.rs [hunk 2]
```diff
@@ -27,10 +27,10 @@ const fn get_max_txbody_size() -> usize {
 }
 
 /// Prefix for the reveal transaction ids - batch proof namespace.
-pub const REVEAL_BATCH_PROOF_PREFIX: &[u8] = get_reveal_batch_proof_prefix();
+pub const TO_BATCH_PROOVER_PREFIX: &[u8] = get_reveal_batch_proof_prefix();
 
 /// Prefix for the reveal transaction ids - light client namespace.
-pub const REVEAL_LIGHT_CLIENT_PREFIX: &[u8] = get_reveal_light_client_prefix();
+pub const TO_LIGHT_CLIENT_PREFIX: &[u8] = get_reveal_light_client_prefix();
```
Comment (eyusufatik) [inline crates/primitives/src/constants.rs]: we can use *_PROOF_PREFIX, not prover

**Review by eyusufatik — COMMENTED** [review]
similar change, my bad

**Review by eyusufatik — APPROVED** [review]

**Pull request merged**